### DISCUSSION TITLE
No Index tag on SEO Components

### DISCUSSION THREAD

**Discussion by monzuralam** [wpengine/faustjs#1893] [Q&A]
Recently, I used faustjs on my headless WordPress Project. my development ongoing. So, I want to discourage search engines on my website.

In SEO, Compoenets there are no features about it.

File Location: `components/SEO/SEO.js`

By default index behavior is true, if anyone wishes to noindex then just pass props `false`;

**Comment by theodesp** [comment]
Hey @monzuralam. Is there a particular issue you are facing here? Thanks

**Comment by monzuralam** [comment]
Hey @theodesp, I can fix it on my current project by modifying the SEO Component. But I think it should be updated globally.
In `WordPress`, we can add an index & no index tag on any page/post. `Faustjs` is a WordPress Headless framework, We should update it on the SEO component & smoothly manage the `index` & `no index` tag.

Currently, in this framework, we can't manage it.

**Comment by theodesp** [comment]
Hey @monzuralam . Thanks for the report. Would you be able to submit a PR about it?

**Comment by mindctrl** [comment]
Ideally `noindex` would be tied to the "Discourage search engines from indexing this site" setting under Settings > Reading in wp-admin. It looks like WPGraphQL does not expose this setting under `readingSettings`. I think we need to open an issue/PR upstream before we can support it here.

**Comment by theodesp** [comment]
cc @jasonbahl

**Comment by jasonbahl** [comment]
I'd recommend using `register_graphql_field` in the meantime to add settings like this to the schema.

**Comment by theodesp** [comment]
Hey @monzuralam thank you for the question. We are going to move this ticket into the discussions section and you can review @jasonbahl suggestion as well.
